### ISSUE TITLE
Semantic Segmentation of Flickr Landscape Images

### ISSUE THREAD

**Issue by GYengera** [NVlabs/SPADE#56]
Hello, 

I had a question regarding which pretrained DeepLab model was used for computing segmentation maps for the Flickr landscape images. The DeepLab model trained on the Pascal VOC dataset appears to label most of the landscape images as background. So was a model that was pretrained on another dataset utilized or did you train a new semantic segmentation  model on landscape images?

**Comment by LorenzoMauro** [comment]
Hi, @GYengera

If you give a look under the live demo http://nvidia-research-mingyuliu.com/gaugan You can find the link to the implementation used: https://github.com/kazuto1011/deeplab-pytorch.

From that page, it seems like the network was trained also on COCO-Stuff which has some outdoor segmentation labels
